README.md

```diff
@@ -39,6 +39,18 @@ For each part of the pipeline, add name of the file as argument to make (for mor
  - `make` or `make data/references.jsonl` runs and merges pipe3 and pipe4
 
 
+## Running a copy of the game locally
```

Comment:
I would put this as an h3 under "Making changes to the wikidata game" as it also eliminates the composer dependencies and such, no?

Reply:
I think you're right; it should move under there. However, composer still needs to be run as far as I can tell either way